server_release_template/web_gui/js/app/nls/settings.js

```diff
@@ -1,98 +1,173 @@
 define({
```

Comment:
I prefer not to add translations for most of advanced settings. They are meant to be modifiable only by users familiar with the codebase. Also they are more likely to change name or behavior.

Reply:
I understand those option can change a lot but some translation already had something.
Instead of having half translated and half not translated i prefers to have everything in the translation files.
Also i think the different users a more a niche users for the moment but not all can understand the English term.
If there is other change i will try to updates the files as much as possible.